bootstrap.sh

```diff
@@ -31,9 +58,9 @@ if [ ! -f killrvideo_bootstrapped ]; then
   # If requested, create a new superuser to replace the default superuser
   if [ "$KILLRVIDEO_CREATE_ADMIN_USER" = 'true' ]; then
     echo "=> Creating new superuser $KILLRVIDEO_ADMIN_USERNAME"
-    cqlsh $dse_ip 9042 -u $admin_user -p $admin_password -e "CREATE ROLE $KILLRVIDEO_ADMIN_USERNAME with SUPERUSER = true and LOGIN = true and PASSWORD = '$KILLRVIDEO_ADMIN_PASSWORD'"
+    cqlsh $dse_ip 9042 -u $admin_user -p $admin_password $dse_ssl $dse_request_timeout -e "CREATE ROLE $KILLRVIDEO_ADMIN_USERNAME with SUPERUSER = true and LOGIN = true and PASSWORD = '$KILLRVIDEO_ADMIN_PASSWORD'"
```

Comment:
These cqlsh commands start to look really messy with all of the options. If we're just repeating the same options over and over, why not just create one variable with all of the options, like "$cqlsh_options" or something like that?

Reply:
Because they are both optional, but...I see your point.  Even optional we could still combine them into a single variable. I honestly hadn't thought about doing that as I like to be explicit.